CHANGES.md

```diff
@@ -15,6 +15,7 @@
   `Dockerfile` from `setup.py` to `pyproject.toml`. Additionally, updated the relevant 
   documentation in `doc/source` to reflect this change from `setup.py` to
   `pyproject.toml.` (related to #992) 
+* Normalisation fails when chunk encoding must be updated (#1033)
```

Comment:
Please be more precise. What normalisation? (Name the respective function.)

Reply:
Better now?